src/components/renderer/renderer.tsx

```diff
@@ -23,6 +24,7 @@ type State = Readonly<typeof defaultState>;
 class Editor extends React.PureComponent<Props, State> {
   public static pathname: string;
   public readonly state: State = defaultState;
+  public unlisten: any;
```

Comment:
Please add a type

Reply:
Do we have a type for it? Should I just use `() => void`

Reply:
Also , we do not use @types/react-router-dom in the app

Reply:
Please include it then.

Reply:
I added it to master.